scene/resources/default_theme/default_theme.cpp

```diff
@@ -828,13 +839,20 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	theme->set_stylebox("tab_selected", "TabBar", style_tab_selected);
 	theme->set_stylebox("tab_unselected", "TabBar", style_tab_unselected);
 	theme->set_stylebox("tab_disabled", "TabBar", style_tab_disabled);
+	theme->set_stylebox("vertical_tab_selected", "TabBar", style_vertical_tab_selected);
+	theme->set_stylebox("vertical_tab_unselected", "TabBar", style_vertical_tab_unselected);
+	theme->set_stylebox("vertical_tab_disabled", "TabBar", style_vertical_tab_disabled);
 	theme->set_stylebox("button_pressed", "TabBar", button_pressed);
 	theme->set_stylebox("button_highlight", "TabBar", button_normal);
 
 	theme->set_icon("increment", "TabBar", icons["scroll_button_right"]);
 	theme->set_icon("increment_highlight", "TabBar", icons["scroll_button_right_hl"]);
 	theme->set_icon("decrement", "TabBar", icons["scroll_button_left"]);
 	theme->set_icon("decrement_highlight", "TabBar", icons["scroll_button_left_hl"]);
+	theme->set_icon("vertical_increment", "TabBar", icons["scroll_button_down"]);
+	theme->set_icon("vertical_increment_highlight", "TabBar", icons["scroll_button_down_hl"]);
+	theme->set_icon("vertical_decrement", "TabBar", icons["scroll_button_up"]);
+	theme->set_icon("vertical_decrement_highlight", "TabBar", icons["scroll_button_up_hl"]);
```

Comment:
Similar code has to be put into `editor_themes.cpp`, else vertical mode can't be used correctly in the editor.